src/app/action-items/action-items.component.ts

```diff
@@ -57,7 +58,36 @@ export class ActionItemsComponent implements OnInit {
       .subscribe(item => {
         if (!!item) {
           this.retrieveActionItems();
+          const justAddedItemId = parseInt(this.justAddedItem.id) + 1;
+          this.router.navigate(['/items'], { queryParams: { id: justAddedItemId } });
         }
       });
   }
+  private getActionItem(itemId: string | undefined): void {
+    this.isLoadingActionItem = true;
+    this.tasksService
+      .getActionItem(itemId)
+      .pipe(first())
+      .subscribe(
+        item => {
+          this.isLoadingActionItem = false;
+          this.actionItem = item;
+        },
+        err => {
+          if (err.status === 404) this.errorMessage = err.statusText;
```

Comment:
**SUGGESTION:** It would be good to add a generic message for the other cases than `404` - eg. `Ooops! Something went wrong`.